### DISCUSSION TITLE
WebSocketsClient still reconnects after calling .disconnect()

### DISCUSSION THREAD

**Discussion by KSDaemon** [khoih-prog/WebSockets_Generic#48] [Q&A]
### Describe the bug

Can't find any info in the docs. So asking here:
There is `.setReconnectInterval()` method for specifying a reconnection interval. And there is a `.disconnect()` method.

I was assuming that automatic reconnection happens only on network errors. And if the client explicitly calls `.disconnect()` - no reconnections should occur after that. But for now, the client is still reconnecting.

Is this an expected behaviour? Or it is a bug? And how to disconnect?

**Comment by khoih-prog** [comment]
Please test on ESP32/ESP8266. If the same issue / behaviour happens, post the issue on  [**arduinoWebSockets**](https://github.com/Links2004/arduinoWebSockets)

**Reply by KSDaemon** [reply]
I'm just having it on ESP32. I then checked with the clean arduinoWebSockets client. And get the same results. So I'll file a bug in the arduinoWebSockets repo. Thnx!

**Reply by khoih-prog** [reply]
Good to know you'll help. 

Please post the link of your issue so that we, me including, can benefit from that.
